crates/rspack_plugin_runtime/src/runtime_module/runtime/expose_global.js

```diff
@@ -0,0 +1,13 @@
+var globalName = "$GLOBAL$";
+var uniqueName = $UNIQUE_NAME$;
+var global = __webpack_require__.g;
+if (global[globalName] && typeof global[globalName] !== "object") {
+  console.warn("[RSPACK] Expose global failed: Global variable `" + globalName + "` already exists and its type is not `Object`.");
```

Comment:
Maybe we should avoid injecting `console` to the output, some JS runtime may not provide `console` object.

Reply:
I prefer to inject minimal runtime code to include Rspack information to avoid affecting the user's bundle size and code readability.